package.json

```diff
@@ -2422,6 +2423,11 @@
                 "title": "%vscode-docker.commands.containers.stop%",
                 "category": "%vscode-docker.commands.category.dockerContainers%"
             },
+            {
+                "command": "vscode-docker.containers.stats",
+                "title": "%vscode-docker.commands.containers.stats%",
+                "category": "%vscode-docker.commands.category.docker%"
```

Comment:
This is intentionally `%vscode-docker.commands.category.docker%` instead of `%vscode-docker.commands.category.dockerContainers%`. I thought it looked better if the command name in the palette was "Docker: Stats" rather than "Docker Containers: Stats".